drivers/Serial.c

```
@@ -37,8 +38,43 @@
+    return size;
+}
+
+static CharDevice_t g_serial_device = {
+    .name = "Serial",
+    .Read = SerialDevRead,
+    .Write = SerialDevWrite,
+};
+
 int SerialInit(void) {
-    return SerialInitPort(COM1);
+    int result = SerialInitPort(COM1);
```

Clang (14.0.6)

[warning] variable 'result' is not initialized

(cppcoreguidelines-init-variables)